src/data/docs/tour/README.md

```diff
@@ -43,7 +43,8 @@ We __do__ need something else to make it nice to work with content-addressed cod
 
 When first launching Unison in a new directory, we get a message like:
 
-> No codebase exists here so I'm initializing one in: .unison/v2
+> No codebase exists in </your/home/directory>.
+> Run `ucm init` to create one there, then try again; or `ucm -codebase <dir>` to load a codebase from someplace else!
 
 What's happening here? This is the Unison Codebase Manager starting up and initializing a fresh codebase. We're used to thinking about our codebase as a bag of text files that's mutated as we make changes to our code, but in Unison the codebase is represented as a collection of serialized syntax trees, identified by a hash of their content and stored in a collection of files inside of that `.unison/v2` directory.
```

Comment:
Could you revert this change; I agree it looks like it needs an adjustment, but this change won't match the following paragraph.  We can try to address it in a separate PR.

Reply:
No problem, just wanted to be precise about the message we get which is not exactly what I saw.